modulefiles/tasks/hera/mm_run.local.lua

```diff
@@ -0,0 +1,2 @@
+unload("python_srw")
+load("python_srw_aqm_eval")
```

Comment:
The `mm_run_*` tasks are failing on Hera with the following error message:
```
ImportError: /contrib/spack-stack/installs/gnu/13.3.0/lib64/libstdc++.so.6: version `CXXABI_1.3.15' not found (required by /scratch3/NCEPDEV/stmp/Michael.Lueken/ufs-srweather-app/hera/conda/envs/aqm-eval/lib/python3.11/site-packages/pyproj/../../../libproj.so.25)
```
Please add `unsetenv("LD_PRELOAD")` to the task modulefile to correct this behavior.

```suggestion
unload("python_srw")
unsetenv("LD_PRELOAD")
load("python_srw_aqm_eval")
```